BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -543,16 +543,18 @@ LocateComponentEntry (
 
   // Locate the component from the container header
   ContainerHdr = (CONTAINER_HDR *)(UINTN)ContainerEntry->HeaderCache;
-  CompEntry = LocateComponentEntryFromContainer (ContainerHdr, ComponentName);
-  if (CompEntry == NULL) {
-    return EFI_NOT_FOUND;
+  if (ComponentName != 0) {

```

Comment:
why do you need to change this ?
When component name is 0,  we return EFI_NOT_FOUND, it is expected.

Reply:
I understand you might want to extend this function to return container header only ?
If so,  I am fine with that.

Reply:
Yes, I would be limited to container header when component is 0.